<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 32, 32);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Result_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="History_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sign">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_percent">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_inv">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>1/x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_pow">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>xÂ²</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_0">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_CE">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_sqrt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>âx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_point">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
}

QPushButton:hover {
	background-color: rgb(50, 50, 50);
}

QPushButton:pressed {
	background-color: rgb(40, 40, 40);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_C">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_minus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>â</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_mult">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>Ã</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_plus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_div">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>Ã·</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_eq">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_back">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(50, 50, 50);
}

QPushButton:hover {
	background-color: rgb(60, 60, 60);
}

QPushButton:pressed {
	background-color: rgb(50, 50, 50);
}</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
